src/main/webapp/app/entities/flair-bi/filter/filter-pane.component.js

```diff
@@ -94,8 +94,17 @@
         }
 
         function addFilterInIframeURL() {
-            var filters = filterParametersService.getSelectedFilter();
-            filterParametersService.setFilterInIframeURL(filters,vm.iframes,vm.dimensions);
+            Views.getCurrentEditState({
+                id: $stateParams.id
```

Comment:
This is duplicate function Views.getCurrentEditState. Place it in angular service